test_fms/mpp/test_mpp_domains2.sh

```diff
@@ -36,10 +36,10 @@ then
 fi
 
 echo "1: Test simple functionality"
+touch input.nml
```

Comment:
I think you should move this above the `echo` statement and add a comments that this is needed before any tests are run